monitoring/prober/scd/test_operation_simple_v0_3_17.py

```diff
@@ -210,6 +210,22 @@ def test_get_op_by_search_latest_time_included(ids, scd_api, scd_session):
   assert ids(OP_TYPE) in [x['id'] for x in resp.json()['operational_intent_references']]
 
 
+@default_scope(SCOPE_AA)
+@depends_on(test_create_op)
+def test_set_uss_availability(ids, scd_session2):
```

Comment:
These two tests verify that the endpoints produce 200 responses, but I think we probably want to make tests a bit more rigorous than that.  After we have a placeholder implementation that does nothing upon set and always returns "Unknown" upon get (step 2 in #612), both of these tests will pass.  Ideally, we want tests that will not pass until the implementation is correct (but then comment out the assertions during development).  To check that, we should at least set a particular USS's availability to each of the possible states and verify that it is that state (via a get) after each set.  Another good test would probably be to verify that uninitialized USSs return Unknown -- to do that, we would get the availability for a USS that we know has never be set, and picking the USS ID using a UUID would probably work well for that.  Another test we might consider is one where we look at two different USS IDs and verify that changing the availability on one doesn't change the availability of the other.

Reply:
Updated with expected result.